cluster/gce/config-default.sh

```diff
@@ -96,9 +96,9 @@ CLUSTER_IP_RANGE="${CLUSTER_IP_RANGE:-$(get-cluster-ip-range)}"
 MASTER_IP_RANGE="${MASTER_IP_RANGE:-10.246.0.0/24}"
 
 if [[ "${FEDERATION:-}" == true ]]; then
```

Comment:
Since this file sources config-common.sh this whole block should just be removed.